.github/workflows/CI.yml

```diff
@@ -66,11 +66,13 @@ jobs:
     - name: Install Protoc
       uses: arduino/setup-protoc@v1
       with:
-        repo-token: ${{ secrets.GITHUB_TOKEN }} 
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
     - uses: Swatinem/rust-cache@v1
     - uses: actions/checkout@master
     - name: Run tests
       run: cargo test --all --all-features
+    - name: Run ignored tonic-health bootstrap test
+      run: cargo test --all-features --package tonic-health --test bootstrap bootstrap -- --ignored --exact
```

Comment:
I don't think this is supposed to be ignored? We want the test to run and it should generate the exact code that was committed? Does it not work like that?

Reply:
`cargo test --ignored` run tests which are marked as `#[ignored]`.
So scheme is next:
1) running `test --all --all-features` in root directory with all packages causes to `transport` feature on for `tonic-builds`, which lead to overriding `dev-dependencies` in `tonic-health`, which lead to rerun `bootstrap.rs` with `transport` feature, which lead to generating `transport` code;
2) so we ignore that `bootstrap` in `test --all`, but run this test separately;

Reply:
I see what you're saying, I think the better approach here is to add a `build_transport` feature that when `transport` is enabled for build can disable it. So in the bootstrap test we can just tell it even if transport is enabled don't generate it. Here is the PR https://github.com/hyperium/tonic/pull/1130 so once that is merged you can just add the `build_transport(false)` setting to `tonic-health`'s bootstrap.rs and then remove the transport feature. Or if it works you can make it so that when health's transport feature is enabled it sets that to true etc.

Reply:
> Ok I finally got the PR merged after 3 days of trying to fix a CI bug with it! So if you update this PR with that then we can move forward. We should back out the CI changes etc the test should pass now.

Okey, now it have:
```
Run cargo test --all --all-features
    Updating git repository `[https://github.com/tokio-rs/prost/`](https://github.com/tokio-rs/prost/%60)
error: failed to load source for dependency `prost-build`

Caused by:
  Unable to update https://github.com/tokio-rs/prost/?branch=lucio/format
  ```